src/js/providers/tracks-mixin.js

```diff
@@ -351,6 +359,20 @@ define(['../utils/underscore',
         }
     }
 
+    function _cancelXhr(itemTracks) {
+        _.each(itemTracks, function(itemTrack) {
+            var xhr = itemTrack.xhr;
+            if (xhr) {
+                xhr.onload = null;
+                xhr.onreadystatechange = null;
+                xhr.onerror = null;
+                if ('abort' in xhr) {
+                    xhr.abort();
+                }
+            }
+        });
+    }
```

Comment:
nope. we need one. This is it. It just needs to be exposed publicly in utils: https://github.com/jwplayer/jwplayer/search?utf8=%E2%9C%93&q=ajax+abort